nixos/modules/virtualisation/hcloud-config.nix

```diff
@@ -0,0 +1,131 @@
+# Configuration for Hetzner Cloud virtual servers.
+
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.hcloud;
+  dynamicHostname = config.networking.hostName == "";
+in
+
+{
+  imports = [
+    ../profiles/qemu-guest.nix
+  ];
+
+  options = {
+    hcloud = {
+      efi = lib.mkOption {
+        default = pkgs.stdenv.hostPlatform.isAarch64;
+        defaultText = lib.literalExpression "pkgs.stdenv.hostPlatform.isAarch64";
+        internal = true;
+        description = ''
+          Whether the server is using EFI to boot.
+        '';
+      };
+      fetchMetadata = lib.mkOption {
+        default = false;
+        description = ''
+          Whether to make server metadata available as
+          {file}`/run/hcloud-metadata` and {file}`/run/hcloud-userdata`.
+        '';
+      };
+    };
+  };
+
+  config = {
+
+    boot.growPartition = true;
+
+    fileSystems."/" = {
+      device = "/dev/disk/by-label/nixos";
+      fsType = "ext4";
+      autoResize = true;
+    };
+
+    fileSystems."/boot" = lib.mkIf cfg.efi {
+      device = "/dev/disk/by-label/ESP";
+      fsType = "vfat";
+      options = [ "umask=0077" ];
+    };
+
+    boot.loader = {
+      timeout = 1;
```

Comment:
1 second is really short. Why not 3 or just keep the default 5?

Reply:
I was following the EC2 config, but it looks like it's just OpenStack en EC2 that do this. I'll remove it.